package.json

```diff
@@ -115,6 +115,7 @@
     "prop-types": "^15.8.1",
     "react-dates": "21.8.0",
     "react-inlinesvg": "^3.0.1",
+    "react-is": "^16.9.0",
```

Comment:
Do we need the types also?
https://www.npmjs.com/package/@types/react-is
or there's no errors shown?

Reply:
Tried doing that, but there is not @types/react-is with the exact same version `^16.9.0`, so I thought it's ok, if I'll just declare it in `types/packages.d.ts` - check it out

Reply:
you don't need the same version, you can use the latest that exists for 16.x, it means there were no significant changes that required upgrading versions

Reply:
Ok, installed `"@types/react-is": "^16.7.5"`👍